Jenkinsfile

```diff
@@ -103,14 +103,16 @@ pipeline {
 						label 'docker&&linux'
 					}
 					environment {
-						GITHUB_API = credentials('github-token')
+						GITHUB_PRIVATE_KEY_B64 = credentials('githubapp-jenkins-infra-reports-private-key-b64')
+						GITHUB_APP_IDENTIFIER = credentials('githubapp-jenkins-infra-reports-app-identifier')
+						GITHUB_ORG_NAME = "jenkinsci"
```

Comment:
While this feels like the correct thing to do, I'd not do this to not accidentally copy it for another org for which it is not true that there are only public repos.

Reply:
I've added `privacy: PUBLIC` l96 to avoid this potential mistake.

Reply:
I think Daniels point was to not do that, there’s no added benefit and only risk if it gets copied elsewhere

Reply:
It served me for my tests but you are right, no point in keeping it now.

Reply:
`privacy: PUBLIC` looks like a good solution 👍